styleguide/source/assets/scss/01-atoms/_table.scss

```diff
@@ -70,18 +70,20 @@ tbody {
       display: table-row;
     }
 
-    &.odd,
-    &:nth-child(odd)  {
-      background: $gray-7;
-    }
-
     td {
       border: 1px solid $gray-7;
       display: block;
-      padding: 2px 2px 2px 50%;
+      padding: 10px;
       position: relative;
+      -webkit-hyphens: auto;
+      -ms-hyphens: auto;
       hyphens: auto;
 
+      &.odd,
+      &:nth-child(odd)  {
```

Comment:
Changing this background from odd `tr` to odd `td` is creating the effect of column striping versus row striping.  Is this intentional?

Reply:
I don't think the column striping in the desktop/tablet viewport is intentional. I can take a look. 

CC: @designerneil Do you have any thoughts on the background color for desktop/tablet viewport? 

The column stacking was decided to accommodate for lengthy title for mobile. It was a two-column before and text was getting cut off and overlapping. See below:

![image](https://user-images.githubusercontent.com/397902/48162387-8c3a2400-e2a2-11e8-8d88-5e5d2547c929.jpg)